src/Jellyfin.Plugin.Dlna/PlayTo/PlayToController.cs

```diff
@@ -846,12 +879,9 @@ private static long GetLongValue(IReadOnlyDictionary<string, string> values, str
     /// <inheritdoc />
     public Task SendMessage<T>(SessionMessageType name, Guid messageId, T data, CancellationToken cancellationToken)
     {
-        if (_disposed)
-        {
-            throw new ObjectDisposedException(GetType().Name);
-        }
-
-        return name switch
+        return _disposed
+            ? throw new ObjectDisposedException(GetType().Name)
+            : name switch
```

Comment:
Is this linter recommended?
I personally don't think this makes the code easier to read.